Project.toml

```
@@ -47,7 +47,7 @@ DocStringExtensions = "0.8"
 DynamicPPL = "0.11.0"
 EllipticalSliceSampling = "0.4"
 ForwardDiff = "0.10.3"
-Libtask = "0.4, 0.5"
+Libtask = "< 0.5.2"
```

That would also allow 0.3 etc. Maybe we can just make it
```suggestion
Libtask = "0.4 - 0.5.1"
```
?

---

Haha, I just had that but it requires Julia 1.4 or higher, so I had to revert it.

---

Ah I see. But I guess equality specifiers might work?
```suggestion
Libtask = "= 0.4.0, = 0.4.1, = 0.4.2, = 0.5.0, = 0.5.1"
```